Unix/xmlserializer/xmlserializer.c

```diff
@@ -890,7 +890,7 @@ static void WriteBuffer_Char(
     };
 
     if ((escapingDepth == 0) ||
-        (charToWrite > L'>') ||
+        (((unsigned int)charToWrite) > L'>') ||
```

Comment:
The length of 'unsigned int' is two or 4 bytes. Consider using wchar_t instead. It is also of variable lenght but it will be the same size as `L'<'`